README.md

```diff
@@ -217,15 +215,20 @@ MIDDLEWARE:
 
 TBD
 
+
+## Overview of the Giftless workflow
```

Comment:
This is the only bit that matters: the rest is just about removing superfluous whitespace.

Reply:
@sglavoie Looks good to me!